src/components/EnergyConsumptionChart.tsx

```diff
@@ -86,7 +86,7 @@ const EnergyConsumptionChart: React.FC = () => {
       breakpoint: smallBreakpoint,
     },
     {
-      name: "Gaming",
+      name: "Gaming in the US",
```

Comment:
This wraps to three lines... small suggestion if we want to compress a little, but I'm fine with it either way
```suggestion
      name: "Gaming (USA)",
```

Reply:
sure - this is similar to my original but @minimalsm suggested it might not be a smooth for translators. Happy either way.